src/fides/api/ops/service/connectors/erasure_email_connector.py

```diff
@@ -104,6 +105,18 @@ def batch_email_send(self, privacy_requests: Query) -> None:
                 batched_identities.extend(filtered_user_identities.values())
             else:
                 skipped_privacy_requests.append(privacy_request.id)
+                ExecutionLog.create(
+                    db=db,
+                    data={
+                        "connection_key": self.configuration.key,
+                        "dataset_name": self.configuration.name,
+                        "collection_name": self.configuration.name,
+                        "privacy_request_id": privacy_request.id,
+                        "action_type": ActionType.erasure,
+                        "status": ExecutionLogStatus.skipped,
+                        "message": f"Erasure email skipped for '{self.configuration.name}'",
```

Comment:
While I'm here, also adding skipped execution logs for erasure email connector to keep this the same.